docs/_docs/memory-configuration/data-regions.adoc

```diff
@@ -84,51 +84,103 @@ tab:C++[unsupported]
 
 == Cache Warm-Up Strategy
 
-The cache warm-up strategy provides an ability to load data from the disk to the node before it is joined to the cluster.
-This prevents performance loss when the node is restarted.
+Ignite does not require you to warm memory up from disk on restarts. As soon as a cluster is inter-connected, your application can query and compute on it. At the same time, the memory warm-up feature is designed for low-latency applications that prefer data being loaded in memory before it can be queried.
 
-[NOTE]
-====
-The cache warm-up strategy is applicable only to the data regions. It can be configured both for all regions (by default) or for each region separately.
-====
-
-Presently, Ignite warm-up strategy implies loading data into the region, starting with indexes, until it runs out of free space.
+Presently, the Ignite warm-up strategy implies loading data into all or specific data regions of Ignite, starting with indexes, until it runs out of free space. It can be configured both for all regions (by default) or for each region separately.
```

Comment:
I would say that at the moment there is one strategy that works as you described.

Reply:
Well, I'm not adding info on any other strategy (as we don't have one).